common/source/docs/common-tarot-gimbal.rst

```diff
@@ -4,10 +4,8 @@
 Tarot Gimbal
 ============
 
-The `Tarot 2-Axis Gimbal <https://store.3dr.com/products/tarot-t-2d-brushless-gimbal-kit>`__
-is the recommended gimbal for the :ref:`Pixhawk <common-pixhawk-overview>`
-and :ref:`APM2 <common-apm25-and-26-overview>` due to its low cost and good
-performance.
+The `Tarot 2-Axis Gimbal <https://www.amazon.com/Tarot-T-2D-Brushless-Gimbal-Robotics/dp/B00OJFWAWS/ref=pd_lpo_sbs_21_t_0?_encoding=UTF8&psc=1&refRID=307GA49DPT5W5VJB7H32>`__
```

Comment:
So this is a link to same gimbal on Amazon. But if it is produced by 3DR will eventually run out and will be back to square one.

1. Are there other compatible gimbals?
2. Are we still recommending this gimbal?
3. Should we archive it?